examples/grid_example/public/app.tsx

```diff
@@ -296,49 +305,60 @@ export const renderGridExampleApp = (
   return () => ReactDOM.unmountComponentAtNode(element);
 };
 
-const layoutStyles = ({ euiTheme }: UseEuiTheme) => {
-  const gridColor = transparentize(euiTheme.colors.backgroundFilledAccentSecondary, 0.2);
+const customLayoutStyles = ({ euiTheme }: UseEuiTheme) => {
   return css({
-    // background for grid row that is being targetted
-    '.kbnGridRow--targeted': {
-      backgroundPosition: `top calc((var(--kbnGridGutterSize) / 2) * -1px) left calc((var(--kbnGridGutterSize) / 2) * -1px)`,
-      backgroundSize: `calc((var(--kbnGridColumnWidth) + var(--kbnGridGutterSize)) * 1px) calc((var(--kbnGridRowHeight) + var(--kbnGridGutterSize)) * 1px)`,
-      backgroundImage: `linear-gradient(to right, ${gridColor} 1px, transparent 1px), linear-gradient(to bottom, ${gridColor} 1px, transparent 1px)`,
-      backgroundColor: `${transparentize(euiTheme.colors.backgroundFilledAccentSecondary, 0.1)}`,
-    },
-    // styling for the "locked to grid" preview for what the panel will look like when dropped / resized
-    '.kbnGridPanel--dragPreview': {
-      borderRadius: `${euiTheme.border.radius}`,
-      backgroundColor: `${transparentize(euiTheme.colors.backgroundFilledAccentSecondary, 0.2)}`,
-      transition: `opacity 100ms linear`,
-    },
-    // styling for panel resize handle
-    '.kbnGridPanel--resizeHandle': {
-      opacity: '0',
-      transition: `opacity 0.2s, border 0.2s`,
-      borderRadius: `7px 0 7px 0`,
-      borderBottom: `2px solid ${euiTheme.colors.accentSecondary}`,
-      borderRight: `2px solid ${euiTheme.colors.accentSecondary}`,
-      '&:hover, &:focus': {
-        outlineStyle: `none !important`,
-        opacity: 1,
-        backgroundColor: `${transparentize(euiTheme.colors.accentSecondary, 0.05)}`,
-      },
-    },
+    // removes the extra padding that EuiPageTemplate adds in order to make it look more similar to Dashboard
+    marginLeft: '-24px',
```

Comment:
is this intentional that these values are not coming from euiTheme?

Reply:
This is just for the example app, and we're not too strict about using EUI, i18n, etc. for these cases. I can switch it to EUI theme vars for consistency, though :)

Reply:
Done in dbdfa8f878b9ef2628c98a85f3aea0b62a429d41 :+1: